pypeit/spectrographs/vlt_xshooter.py

```diff
@@ -221,11 +221,13 @@ def default_pypeit_par(self):
         par['calibrations']['wavelengths']['ech_sigrej'] = 3.0
 
         # Flats
-        par['calibrations']['standardframe']['process']['illumflatten'] = False
-        par['scienceframe']['process']['illumflatten'] = False
+        #par['calibrations']['standardframe']['process']['illumflatten'] = False
         par['calibrations']['flatfield']['tweak_slits_thresh'] = 0.90
         par['calibrations']['flatfield']['tweak_slits_maxfrac'] = 0.10
 
+        # Turn of illumflat
+        turn_off = dict(use_illumflat=False, use_biasimage=False, use_overscan=False)
+        par.reset_all_processimages_par(**turn_off)
 
```

Comment:
Where are you requiring dark subtraction for the flats? You should be doing that here???

Reply:
If you think they are *required*.  Then move the block in the NIR Dev suite pypeit file
into here.

Reply:
They are required for NIR. The stuff you commented out was implementing that. I've put it back in the spectrograph file I think, but have a look to check it is correct.

Reply:
looks good to me.